### PR TITLE
Removes check to see if a post exists and adds logging 

### PR DESCRIPTION
**Pull request by chloealee**
#### What's this PR do?
- Removes check to see if a post exists in Rogue (dupes will be on the data reporting side, see convo [here](https://dosomething.slack.com/archives/C2BLZGBPH/p1553524974012200)).
- Adds logging if an action doesn't exist for a CallPower campaign id.

#### How should this be reviewed?
👀 

#### Relevant tickets

Fixes some code from #64

### CODE REVIEW

**Comment by chloealee** [comment]
Flagging for @mjain-1 too that we won't be checking Rogue to see if a post exists by `northstar_id` and `action_id` (what we discussed in the Tech Spec)!

File: app/Jobs/CreateCallPowerPostInRogue.php [hunk 1]
```diff
@@ -43,32 +43,24 @@ public function handle(Rogue $rogue)
         // Using the callpower_campaign_id, get the action id from Rogue.
         $actionId = $rogue->getActionIdFromCallPowerCampaignId($this->parameters['callpower_campaign_id']);
 
-        // Check if the post exists in Rogue. If not, create the post.
-        $existingPost = $rogue->getPost([
-          'action_id' => $actionId,
-          'northstar_id' => $user->id,
+        info('creating post in rogue for northstar user: ' . $user->id);
```
Comment (katiecrane) [inline app/Jobs/CreateCallPowerPostInRogue.php]: Would it be helpful if this log included that this was coming from the CallPower job?

**Comment by katiecrane** [comment]
@chloealee I wanted to confirm that CallPower won't be sending over any true duplicates?

File: app/Jobs/CreateCallPowerPostInRogue.php (hunk 1 above)
Comment (chloealee) [reply app/Jobs/CreateCallPowerPostInRogue.php]: @katiecrane added the details!

**Comment by chloealee** [comment]
@katiecrane more info in this [Slack convo](https://dosomething.slack.com/archives/C2BLZGBPH/p1553524974012200)! Does that answer your question?

**Comment by katiecrane** [comment]
@chloealee ah I see it now! thank you!!

**Pull request merged**